src/styles/base/_typography.scss

```diff
@@ -165,6 +167,7 @@ pre[ class*="language-" ] {
 	border-radius: 2px;
 }
 
-::selection {
+::selection,
+pre::selection {
```

Comment:
Actually this isn't necessary at all now as I just removed the styles from prism.scss. I'll remove it!